analysis/create_project_actions.R

```diff
@@ -431,7 +431,7 @@ actions_list <- splice(
                  "table2_vax",
                  "table2_unvax"),
     moderately_sensitive = list(
-      table2_output_rounded = glue("output/table2_output_rounded.csv")
+      table2_output_midpoint6_derived = glue("output/table2_output_midpoint6_derived.csv")
```

Comment:
There are no specific recommendations around filenames as far as I can see, just variable names. Please pick one filename type and be consistent though so either '_rounded' or '_midpoint6' or '_midpoint6_derived'.

Reply:
Your instructions on slack were unclear, you only requested to change _rounded with _midpoint6 and _midpoint6_derived. The changes I made work, and there has been no issue with dummy data and real data (analysis currently running on the server).